test-requirements.txt

```
@@ -0,0 +1 @@
+ odoo-addon-account_payment_mode @ git+https://github.com/OCA/bank-payment.git@refs/pull/791/head#subdirectory=account_payment_mode
```

```suggestion
 odoo-addon-account_payment_mode @ git+https://github.com/OCA/bank-payment.git@refs/pull/1367/head#subdirectory=account_payment_mode
```
Isn't the correct migration PR of `account_payment_mode` to 18.0 this one? --> #1367